packages/roosterjs-content-model-api/lib/publicApi/utils/getEditorHTMLContent.ts

```diff
@@ -0,0 +1,25 @@
+import { transformColor } from 'roosterjs-content-model-dom';
+import type { IEditor } from 'roosterjs-content-model-types';
+
+/**
+ * Return the inner HTML content of the editor
+ * @param editor The editor object
+ * @returns
+ */
+export function getEditorHTMLContent(editor: IEditor): string {
```

Comment:
Instead of adding a new API, how about just replace the existing code of exportContent when it requests cleanHTML?

Reply:
And @BryanValverdeU I think we just need to use this one from roosterjs directly, and in OWA side we only do the content model output length check and logging, then later when we solve the problem, we can easily remove the override